include/psen_scan_v2/diagnostics.h

```diff
@@ -26,14 +26,18 @@
 
 namespace psen_scan_v2
 {
-static constexpr uint32_t RAW_DIAGNOSTIC_MESSAGE_LENGTH_FOR_ONE_DEVICE_IN_BYTES{ 9 };
-static constexpr uint32_t RAW_DIAGNOSTIC_MESSAGE_UNUSED_OFFSET_IN_BYTES{ 4 };
-static constexpr uint32_t RAW_DIAGNOSTIC_MESSAGE_LENGTH_IN_BYTES{
-  RAW_DIAGNOSTIC_MESSAGE_UNUSED_OFFSET_IN_BYTES +
-  RAW_DIAGNOSTIC_MESSAGE_LENGTH_FOR_ONE_DEVICE_IN_BYTES * VALID_SCANNER_IDS.size()
-};
-
-using RawDiagnosticMsg = std::array<uint8_t, RAW_DIAGNOSTIC_MESSAGE_LENGTH_IN_BYTES>;
+namespace monitoring_frame
+{
+namespace diagnostic
+{
+namespace raw_message
+{
+static constexpr uint32_t LENGTH_FOR_ONE_DEVICE_IN_BYTES{ 9 };
+static constexpr uint32_t UNUSED_OFFSET_IN_BYTES{ 4 };
+static constexpr uint32_t LENGTH_IN_BYTES{ UNUSED_OFFSET_IN_BYTES +
+                                           LENGTH_FOR_ONE_DEVICE_IN_BYTES * VALID_SCANNER_IDS.size() };
+using Type = std::array<uint8_t, diagnostic::raw_message::LENGTH_IN_BYTES>;
+}  // namespace raw_message
 
 enum class DiagnosticCode
```

Comment:
Above it is defined: `raw_message::Type` but here it is defined: `diagnostic::DiagnosticCode`. This is inconsistent. I suggest:
```suggestion
enum class ErrorType
```

Reply:
I prefer changing the name of diagnostic::raw_message::Type to diagnostic::raw_message::Field

Reply:
If in your opinion is the better name then use this name. My point is that the `Diagnostics` should not be included in the name because it is already included in the namespace name.